doc/backmatter/references.rst

```diff
@@ -14,6 +14,5 @@ References
 
    .. _sphinxbib: https://pypi.org/project/sphinxcontrib-bibtex/
 
-.. bibliography:: ../_static/references.bib
-   :all:
```

Comment:
Now I'm only seeing a single entry in the references (the PRM), I guess because it's the only one that's currently both in the bib file and being cited? Will we need to go through and update the other citations in the docs if we want them to appear here?

Reply:
Added a feature request, https://github.com/retorquere/zotero-better-bibtex/issues/3137 . I suppose we could also add a "collection" in zotero that could serve a similar purpose.